plugins/frontend/backstage-plugin-jira/src/api/index.ts

```diff
@@ -296,7 +304,20 @@ export class JiraAPI {
 
       ticketIds = foundIssues.map(i => i.key);
     }
-
+    tickets = foundIssues.map(index => {
+      return {
+        key: index.key,
+        summary: index?.fields?.summary,
+        assignee: {
+          displayName: index?.fields?.assignee?.displayName,
+          avatarUrl: index?.fields?.assignee?.avatarUrls['48x48'],
+        },
+        status: index?.fields?.status?.name,
+        priority: index?.fields?.priority,
+        created: index?.fields?.created,
+        updated: index?.fields?.updated,
+      };
+    }) as any;
```

Comment:
Let's specify a proper type for the `tickets` variable. This return from `map` doesn't need to be cast as `any`

Reply:
Fixed this issue